### PR TITLE
Fix /op Slash Command

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/riot-web/issues/13697
Fixes https://github.com/vector-im/riot-web/issues/12527

Only allows `/op`ing of users which are invited/joined to the room.

### CODE REVIEW

**Comment by aaronraimist** [comment]
I don't understand why you shouldn't be able to set the power level of someone who isn't in the room yet. I guess this is intending to fix https://github.com/vector-im/riot-web/issues/12527 but I agree with https://github.com/vector-im/riot-web/issues/12527#issuecomment-625885467.

**Comment by t3chguy** [comment]
Right sure it'd be better as a warning

File: src/SlashCommands.tsx [hunk 1]
```diff
@@ -729,9 +729,9 @@ export const Commands = [
                         const cli = MatrixClientPeg.get();
                         const room = cli.getRoom(roomId);
                         if (!room) return reject(_t("Command failed"));
-
+                        const member = room.getMember(args);
+                        if (!member || member.membership !== "join") return reject(_t("Could not find user in room"));
```
Comment (turt2live) [inline src/SlashCommands.tsx:733]: this probably needs a product decision as it addresses https://github.com/vector-im/riot-web/issues/12527

Comment (t3chguy) [inline src/SlashCommands.tsx:733]: should also accept `invite`

**Comment by bluecmd** [comment]
Hi folks! Great work on Matrix/Riot! Any news on when this fix can be merged? It's pretty annoying to have to run a separate [Python script](https://gist.github.com/bluecmd/c67d2c84c0021729e74808ad2535bb11) to be able to change people's power levels, so I would greatly appreciate this fix :-).

**Comment by t3chguy** [comment]
> Hi folks! Great work on Matrix/Riot! Any news on when this fix can be merged? It's pretty annoying to have to run a separate [Python script](https://gist.github.com/bluecmd/c67d2c84c0021729e74808ad2535bb11) to be able to change people's power levels, so I would greatly appreciate this fix :-).

You do realise there is a graphical way to set power levels in User Info too right?

image
image

**Comment by bluecmd** [comment]
> You do realise there is a graphical way to set power levels in User Info too right?

I had *no* idea! TIL! Thanks, that's a very nice way.

**Pull request merged**